pysat/_instrument.py

```diff
@@ -2710,18 +2723,17 @@ def to_netcdf4(self, fname=None, base_instrument=None, epoch_name='Epoch',
                             for export_units_label in export_units_labels:
                                 new_dict[export_units_label] = \
                                     'Milliseconds since 1970-1-1 00:00:00'
-                            new_dict = self._filter_netcdf4_metadata(new_dict,
-                                                                     coltype,
-                                                                     export_nan=export_nan)
+                            new_dict = self._filter_netcdf4_metadata(
+                                new_dict, coltype, export_nan=export_nan)
```

Comment:
Using the online tool here it looks like the lower line is indented with only 4 spaces. Isn't this supposed to be avoided when extending a statement across multiple lines?

https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
This comment applies to a number of similar changes to the _filter_netcdf4 calls.

Reply:
The above format is technically correct (at least according to flake8), but may be less clear in readability.  For instance the above is allowed, but the following is not
```
new_dict = self._filter_netcdf4_metadata(new_dict
    coltype, export_nan=export_nan)
```
If I try to allign everything to the paren, it exceeds the 80 character limit.  An alternative would be to use a shorter function name for `_filter_netcdf4_metadata`, or possibly `export_nan`.

Reply:
Doesn't having only 4 spaces for the continuation line make it appear like python whitespace formatting?

Would `_filter_nc_meta` make a difference?

I'm going to be out for a bit at a doctor but I'll be back this afternoon.

Reply:
Still 2 characters too long

Reply:
Although if we also use a line break, it could work:
```
                            new_dict = \
                                self._filter_nc_meta(new_dict,
                                                     coltype,
                                                     export_nan=export_nan)
```

Reply:
Reading over the link @rstoneback posted, I think the original line is fine.  It looks like this is only an issue when defining a new function, not using a function call.  Also, I believe it's preferable to use the paren whitespace breaks than `\`.